lte/gateway/python/magma/enodebd/devices/baicells_qrtb.py

```diff
@@ -343,16 +343,19 @@ def get_msg(self, message: Any) -> AcsMsgAndTransition:
         Returns:
             AcsMsgAndTransition
         """
-        if self.wait_timer.is_done():
-            return AcsMsgAndTransition(
-                msg=models.DummyInput(),
-                next_state=self.done_transition,
+        if self.wait_timer:
+            if self.wait_timer.is_done():
+                return AcsMsgAndTransition(
+                    msg=models.DummyInput(),
+                    next_state=self.done_transition,
+                )
+            remaining = self.wait_timer.seconds_remaining()
+            logger.info(
+                'Waiting with eNB configuration for %s more seconds. ',
+                remaining,
             )
-        remaining = self.wait_timer.seconds_remaining()
-        logger.info(
-            'Waiting with eNB configuration for %s more seconds. ',
-            remaining,
-        )
+        else:
+            logger.warning('wait_timer is None.')
```

Comment:
I think this could be an error and raise a ValueError. (The old behavior was to raise an error in that case too.) Maybe you could invert the logic.
```
if not self.wait_timer:
  logger.error(...)
  raise ValueError(...)
if self.wait_timer.is_done():
  ...
```